src/styles.css

```diff
@@ -0,0 +1,601 @@
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons+Outlined");
```

Comment:
Some changes I'd recommend - 

1. use variables for spacing, color, font, etc
2. use rem, em, vh, vw & % instead of pixels, avoid pixels as much you can.